test/unittests/tracing_test.cpp

```diff
@@ -234,38 +226,36 @@ TEST_F(tracing, trace_revert)
     const auto code = mstore(0, 0x0e4404) + push(3) + push(29) + OP_REVERT;
     trace_stream << '\n';
     EXPECT_EQ(trace(code), R"(
-{"depth":0,"rev":"Berlin","static":false}
-{"pc":0,"op":98,"opName":"PUSH3","gas":0xf4240,"stack":[],"memorySize":0}
-{"pc":4,"op":96,"opName":"PUSH1","gas":0xf423d,"stack":["0xe4404"],"memorySize":0}
-{"pc":6,"op":82,"opName":"MSTORE","gas":0xf423a,"stack":["0xe4404","0x0"],"memorySize":0}
-{"pc":7,"op":96,"opName":"PUSH1","gas":0xf4234,"stack":[],"memorySize":32}
-{"pc":9,"op":96,"opName":"PUSH1","gas":0xf4231,"stack":["0x3"],"memorySize":32}
-{"pc":11,"op":253,"opName":"REVERT","gas":0xf422e,"stack":["0x3","0x1d"],"memorySize":32}
-{"error":"revert","gas":0xf422e,"gasUsed":0x12,"output":"0e4404"}
+{"pc":0,"op":98,"gas":"0xf4240","gasCost":"0x3","memSize":0,"stack":[],"depth":1,"refund":0,"opName":"PUSH3"}
+{"pc":4,"op":96,"gas":"0xf423d","gasCost":"0x3","memSize":0,"stack":["0xe4404"],"depth":1,"refund":0,"opName":"PUSH1"}
+{"pc":6,"op":82,"gas":"0xf423a","gasCost":"0x3","memSize":0,"stack":["0xe4404","0x0"],"depth":1,"refund":0,"opName":"MSTORE"}
+{"pc":7,"op":96,"gas":"0xf4234","gasCost":"0x3","memSize":32,"stack":[],"depth":1,"refund":0,"opName":"PUSH1"}
+{"pc":9,"op":96,"gas":"0xf4231","gasCost":"0x3","memSize":32,"stack":["0x3"],"depth":1,"refund":0,"opName":"PUSH1"}
+{"pc":11,"op":253,"gas":"0xf422e","gasCost":"0x0","memSize":32,"stack":["0x3","0x1d"],"depth":1,"refund":0,"opName":"REVERT"}
 )");
 }
 
-TEST_F(tracing, trace_create)
-{
-    vm.add_tracer(evmone::create_instruction_tracer(trace_stream));
-
-    trace_stream << '\n';
-    EXPECT_EQ(trace({}, 2), R"(
-{"depth":2,"rev":"Berlin","static":false}
-{"error":null,"gas":0xf4240,"gasUsed":0x0,"output":""}
-)");
-}
-
-TEST_F(tracing, trace_static)
-{
-    vm.add_tracer(evmone::create_instruction_tracer(trace_stream));
-
-    trace_stream << '\n';
-    EXPECT_EQ(trace({}, 2, EVMC_STATIC), R"(
-{"depth":2,"rev":"Berlin","static":true}
-{"error":null,"gas":0xf4240,"gasUsed":0x0,"output":""}
-)");
-}
+// TEST_F(tracing, trace_create)
```

Comment:
update test.

Reply:
I think better to remove them. Tracing return empty string now.